apps/aecore/src/aec_db.erl

```diff
@@ -429,6 +428,24 @@ find_tx_location(STxHash) ->
            [#aec_tx_location{value = BlockHash}] -> BlockHash
        end).
 
+-spec find_tx_with_location(binary()) ->
+                                   'none'
+                                 | {'mempool', aetx_sign:signed_tx()}
+                                 | {binary(), aetx_sign:signed_tx()}.
+find_tx_with_location(STxHash) ->
```

Comment:
(Mostly out of scope for this PR so optional for this PR.) Is the [order of the signers documented](https://github.com/aeternity/protocol/blob/414051131a29b808bd829525f773e8fc3ae75d2d/serializations.md#signed-transaction) anywhere to be deterministic?

Reply:
The order is defined in the code, but it is probably not documented.

Reply:
Opened https://github.com/aeternity/protocol/pull/83 with separate Pivotal ticket.